### PR TITLE
308 new app in town

### PR DESCRIPTION
**Pull request by mdartic**
Create the foundation for the new LocoKit app.

Reset the nuxt app to a SPA/PWA app.

### CODE REVIEW

File: api/eslint.config.mjs
Comment (Lonnytunes) [inline api/eslint.config.mjs:1]: Is this `.mjs` extension volontary?

File: packages/vue-components/stories/Welcome.mdx [hunk 1]
```diff
@@ -21,12 +20,11 @@ regarding their props or events.
...
+- secondary (based on the orange of logo)
 
 <div class="bg-primary p-4 m-4 text-2xl flex items-center justify-center text-white">
   Primary color
 </div>
-<div class="bg-secondary p-4 m-4 text-2xl flex items-center justify-center">
-  Secondary color
-</div>
\ No newline at end of file
+<div class="bg-secondary p-4 m-4 text-2xl flex items-center justify-center">Secondary color</div>
```
Comment (Lonnytunes) [inline packages/vue-components/stories/Welcome.mdx:30]: I imagine that it is the linter which performed this change.

If I'm right, could we disable this kind of correction?

That brings more noise than real value in my opinion.

Comment (mdartic) [reply packages/vue-components/stories/Welcome.mdx:30]: Mmm, I don't want to remove this changes, as it allows us to have a consistent base of code.
Nevertheless, in this case, that's my fault as I created this `.mdx` file without applying the linter.
I think that kind of changes won't occur often.

File: app/src/locales/en.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import locokitEn from '@locokit/locales/en.json'
+
+export default {
+  app: {
+    routes: {
+      auth: {
+        login: 'Connection to the back office',
```
[the remark this answers is not in the document]
Comment (mdartic) [reply app/src/locales/en.ts:7]: Do you think it's the same in french ?

File: app/index.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <link rel="apple-touch-icon" href="/apple-touch-icon-180x180.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LocoKit</title>
+  </head>
+  <body class="h-full">
+    <div id="app" class="h-full"></div>
+    <script type="module" src="/src/main.ts"></script>
+    <script>
+      const LOCOKIT_ENV = 'dev'
+    </script>
```
[the remark this answers is not in the document]
Comment (mdartic) [reply app/index.html]: removed and add some settings

File: app/src/composables/useSDK.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import restClient from '@feathersjs/rest-client'
+import { createClient } from '@locokit/sdk'
+
+export function useSDK() {
+  const sdkRestClient = restClient('https://next.locokit.io/api')
```
[the remark this answers is not in the document]
Comment (mdartic) [reply app/src/composables/useSDK.ts]: right, created a new `LCK_SETTINGS` global var.

File: app/src/locales/en.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Lonnytunes) [reply app/src/locales/en.ts]: According to WordReference, there is no dash between the 2 words in English, but Linguee returns examples with and without it.

Personally, I prefer to write it as "one word" by using a dash, whatever the language.

File: api/eslint.config.mjs
Comment (Lonnytunes) [reply api/eslint.config.mjs:1]: I confused it with the `.mdx` extension.

**Review by Lonnytunes — APPROVED** [review]

**Pull request merged**